src/aspnet/4.8.1/windowsservercore-ltsc2022/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# escape=`
+
+ARG REPO=mcr.microsoft.com/dotnet/framework/runtime
+FROM $REPO:4.8.1-20220809-windowsservercore-ltsc2022
+
+RUN dism /Online /Quiet /Enable-Feature /All /FeatureName:IIS-WebServerRole /FeatureName:NetFx4Extended-ASPNET45 /FeatureName:IIS-ASPNET45 `
+    && dism /Online /Quiet /Disable-Feature /FeatureName:IIS-WebServerManagementTools `
+    && del /q "C:\inetpub\wwwroot\*" `
+    && for /D %p IN ("C:\inetpub\wwwroot\*") DO rmdir "%p" /s /q `
+    && curl -fSLo ServiceMonitor.exe https://dotnetbinaries.blob.core.windows.net/servicemonitor/2.0.1.10/ServiceMonitor.exe `
+    && %windir%\Microsoft.NET\Framework64\v4.0.30319\ngen update `
+    && %windir%\Microsoft.NET\Framework\v4.0.30319\ngen update
+
+# Install 2.9.0 Roslyn compilers
+RUN curl -fSLo microsoft.net.compilers.2.9.0.zip https://api.nuget.org/packages/microsoft.net.compilers.2.9.0.nupkg `
+    && mkdir C:\RoslynCompilers `
+    && tar -C C:\RoslynCompilers -zxf microsoft.net.compilers.2.9.0.zip `
+    && del microsoft.net.compilers.2.9.0.zip `
+    && %windir%\Microsoft.NET\Framework64\v4.0.30319\ngen install C:\RoslynCompilers\tools\csc.exe /ExeConfig:C:\RoslynCompilers\tools\csc.exe `
+    && %windir%\Microsoft.NET\Framework64\v4.0.30319\ngen install C:\RoslynCompilers\tools\vbc.exe /ExeConfig:C:\RoslynCompilers\tools\vbc.exe `
+    && %windir%\Microsoft.NET\Framework64\v4.0.30319\ngen install C:\RoslynCompilers\tools\VBCSCompiler.exe /ExeConfig:C:\RoslynCompilers\tools\VBCSCompiler.exe
```

Comment:
These are very old and unsupported compilers. My intuition is that by putting them directly into the `C:\RoslynCompilers\tools` the are _preferred_ over the more recent compilers being installed. Is that the case? If so is there a reason for this cause it seems counter to pushinig people to supported tooling.

Reply:
The aspnet Dockerfiles were originally defined by @HongGit's team which were then merged into this repo. Hopefully @HongGit can provide some info here.